### DISCUSSION TITLE
Email - UI issues in dark mode of Flex

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#2442] [General] [answered]
Hi there,

I have a question regarding the text highlighted colour state for dark theme. Please see the example shared here:

https://user-images.githubusercontent.com/49287701/171400472-0900af75-a745-4bf5-809e-61434973f143.mov

The text in the email body appears black here, although it should be switched to the white version in dark mode example.

I wanted to double check that the dark mode example should work like this example in the Paste website? (See screen recording below)

https://user-images.githubusercontent.com/49287701/171400835-55a59cd3-b066-4850-8bc5-52648c8e8aa0.mov

Also does the highlighted state handle any colour value or is it only designed for Tokens? 

I hope this makes sense, please ping me if not.

Porlin

**Comment by loreina** [comment]
Hi @Porlin, thanks for sharing this! Do you know which Paste component is being used there in Flex? Would it possible to reproduce this in a CodeSandbox so our team can investigate? Thank you!

**Reply by Porlin** [reply]
Hey @loreina just spoke to one of our Engineers and he said he doesn't think it's a Paste issue at all - we need to set the base colour of that iframe. Unfortunately can't reproduce in CodeSandbox.

**Reply by serifluous** [reply]
Hey @Porlin, were you able to share the options below with your engineering team, or have you worked out another solution? Happy to have them looped in if that helps, too!

**Reply by serifluous** [reply]
@Porlin Closing this thread for inactivity. Let us know if you need any more support on this 🙌

**Comment by loreina** [comment, marked as answer]
Hi @Porlin, if you're using Paste's `color-text` token (https://paste.twilio.design/tokens/list#text-colors) for the text, the text color should swap depending on the theme as handled by the browser. 

It's possible to [change the highlight color in CSS](https://css-tricks.com/overriding-the-default-text-selection-color-with-css/) though we would recommend against it as using our tokens would be helpful in catching any updates we make to tokens across themes. Let me know if you have any questions!

**Reply by shleewhite** [reply]
@Porlin another thing they could check is: when the user is in dark mode, they have the CSS flag `prefers-color-scheme` set to `"dark"`. If you have this enabled, the browser has better default styles for UIs in dark mode.

More info here: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
